napari/layers/tracks/_track_utils.py

```diff
@@ -0,0 +1,318 @@
+from typing import Dict
+
+import numpy as np
+from scipy.spatial import cKDTree
+
+from ..utils.layer_utils import dataframe_to_properties
+
+
+def connex(vertices: np.ndarray) -> list:
+    """ make vertex edges for vispy Line """
+    return [True] * (vertices.shape[0] - 1) + [False]
+
+
+class TrackManager:
+    """ TrackManager
+
+    Class to manage the track data and simplify interactions with the Tracks
+    layer.
+
+
+    Properties:
+        data
+        properties
+        points
+
+        track_vertices
+        track_connex
+        track_times
+        track_labels
+
+        graph_vertices
+        graph_connex
+        graph_times
```

Comment:
I think it will be important for future maintenance to have solid docs on these properties.